<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExplorerScannerView</class>
 <widget class="QWidget" name="ExplorerScannerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QLineEdit" name="lineEditTarget"/>
   </item>
   <item>
    <widget class="QToolButton" name="buttonMultipleTargets">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonGet">
     <property name="text">
      <string>Get</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonStop">
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
